_events/2021-1-25-Metabolomics-Mini-Symposium.md

```diff
@@ -0,0 +1,21 @@
+---
+site: freiburg
+tags:
+title: "Galaxy Metabolomics Mini-Symposium"
+starts: 2021-01-29
+ends: 2021-01-29
+organiser:
+  name: Melanie Föll
+  email: foellmelanie@gmail.com
+location: online
+external: https://docs.google.com/document/d/1F3gKicQ7MaMn-EKwC3cJbhf7s8-j4RYtdjEJQH0EeK0/preview
```

Comment:
external link means that only the first paragraph will be previewed, left @bgruening?

Reply:
yes, and only in the preview, that is probably not what we want, do we?

Reply:
yeah so might be better to embed the link in the body